tests/test_community.py

```diff
@@ -48,12 +62,22 @@ def test_community_get(client):
 
     TODO this test needs to be modified when the database is connected
     """
-    rv = client.get(
-        "/community",
-        query_string={"name": "TestCommunity"},
-        follow_redirects=True,
-    )
-    assert b"OK: Community Returned" == rv.data
+    with patch("biit_server.community_handler.Database") as mock_database:
+
+        instance = mock_database.return_value
+
+        query_data = {
+            "name": "TestCommunity",
+        }
+
+        instance.get.return_value = query_data
+
+        rv = client.get(
+            "/community",
+            query_string={"name": "TestCommunity"},
+            follow_redirects=True,
+        )
+        assert query_data == json.loads(rv.data)
```

Comment:
this doesn't test authentication which concerns me since only authenticated users should be able to access communities